app/javascript/components/subject/MonitoringStatus.js

```diff
@@ -162,6 +163,12 @@ class MonitoringStatus extends React.Component {
               ]
             : null,
       });
+    } else if (event?.target?.name && event.target.name === 'apply_to_group') {
+      let applyToGroup = event.target.id === 'apply_to_group_yes' ? true : false;
```

Comment:
Not to nitpick, but the `true`/`false` literals here are unnecessary.

Reply:
done